android/app/src/main/java/io/mosip/residentapp/RNSecureKeystoreModule.java

```diff
@@ -141,7 +150,11 @@ public Unit invoke(Integer code, String message) {
   }
 
   @ReactMethod
-  public void sign(String alias, String data, Promise promise) {
+  public void sign(String signAlgorithm,String alias, String data, Promise promise) {
+    if("RS256".equals(signAlgorithm))
+      signAlgorithm="SHA256withRSA";
+    else
```

Comment:
what happens if signAlgorithm is neither RS256 nor ECDSA?

Reply:
Currently from native side, we have support of this 2 signature only

Reply:
Use if else and handle else scenario